### DISCUSSION TITLE
ansible for support for Authentication Profile , Authentication Sequence and User Identication and ldap server support

### DISCUSSION THREAD

**Discussion by cb1234** [PaloAltoNetworks/pan-os-ansible#217] [Ideas]
i dont see pan ansible module support for above features.  is it possible to add ?

**Comment by jkoolwal** [comment]
+1 to this FR. This is an important functionality missing from the PAN ansible modules.

**Comment by Nothing4You** [comment]
also group mappings (subtype of uder identification)

**Comment by mrichardson03** [comment]
Please see #219 for a way these functions could be added without having to create modules for all of them.
